### ISSUE TITLE
add new VETS data to sheet

### ISSUE THREAD

**Issue by judgelord** [judgelord/correspondence#41]

**Comment by judgelord** [comment]
@rochelle-snyder did we get new DOL_VETS data?

**Comment by rochelle-snyder** [comment]
@judgelord Yes, but some of it is from a different database so it's in a different format. That file is on dropbox here: https://www.dropbox.com/sh/8gnrgyhvv56g5m2/AAAY2j_yYvA-qIX5hb9JLYFKa?dl=0 (and it is the only part of the new data left to be added)

**Comment by judgelord** [comment]
Note: there are duplicate rows for the same letter (same SIMS ID, same date, same author) for letters that were edited on different dates.
